### PR TITLE
38 character selection

### PR DESCRIPTION
**Pull request by chanson02**
Added a screen where the user can select which player they want to play as.

### CODE REVIEW

File: dunwoody_disaster/views/victoryScreen.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+class VictoryScreen(QWidget):
+    def __init__(self, fight_controller: FightSequence):
+        super().__init__()
+        self._callback = DD.unimplemented
+
+        layout = QVBoxLayout()
+        layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+        self.setLayout(layout)
+
+        lbl = QLabel(f"Congradulations {fight_controller.player.name}! You Win!")
```
Comment (NoahBartell) [inline dunwoody_disaster/views/victoryScreen.py]: Congratulations*

File: dunwoody_disaster/views/MapScreen.py [hunk 2]
```diff
@@ -4,35 +4,56 @@
...
+            # self.mapPic.setPixmap(QPixmap(self.imagePaths[self.currImgIndex]))
             print("left")
 
-        elif event.key() == Qt.Key_Right:
-            self.currImgIndex = (self.currImgIndex + 1) % len(self.imagePaths)
-            self.mapPic.setPixmap(QPixmap(self.imagePaths[self.currImgIndex]))
+        elif event.key() == Qt.Key.Key_Right:
+            # self.currImgIndex = (self.currImgIndex + 1) % len(self.imagePaths)
+            # self.mapPic.setPixmap(QPixmap(self.imagePaths[self.currImgIndex]))
             print("right")
```
Comment (NoahBartell) [inline dunwoody_disaster/views/MapScreen.py]: Old prints and comments

**Review by NoahBartell — APPROVED** [review]
Besides a couple of clean up everything looks good to me

File: dunwoody_disaster/views/victoryScreen.py (hunk 1 above)
Comment (chanson02) [reply dunwoody_disaster/views/victoryScreen.py]: Lol good catch. This was from a different PR tho.

File: dunwoody_disaster/views/MapScreen.py (hunk 2 above)
Comment (chanson02) [reply dunwoody_disaster/views/MapScreen.py]: I'm marking this as resolved because it's out of scope for this PR.

**Review by EngineeringDude123 — APPROVED** [review]
Fantabulous

**Review by instMeyFTW — APPROVED** [review]

**Pull request merged**